etc/profile-a-l/android-studio.profile

```diff
@@ -10,11 +10,11 @@ noblacklist ${HOME}/.android
 noblacklist ${HOME}/.jack-server
 noblacklist ${HOME}/.jack-settings
 noblacklist ${HOME}/.local/share/JetBrains
-noblacklist ${HOME}/.ssh
 noblacklist ${HOME}/.tooling
 
 # Allows files commonly used by IDEs
 include allow-common-devel.inc
+include allow-ssh.inc
```

Comment:
Placing: without separation it suggests include allow-ssh.inc is part of the "# Allows files commonly used by IDEs" block. Add the "# Allow ssh (blacklisted by disable-common.inc)" line cfr. your template. We do this all over to avoid confusion/improve readability of our profiles. Same goes for all other profiles where allow-ssh.inc gets included. I'll refer to this remark in the other files to keep things simple.

Reply:
Added the comments, blank lines and also did some sorting:

<https://github.com/netblue30/firejail/compare/2bf2581d6f4c233b84a3d6a475799d2ae5efa013..83ac0239722f85ffed15e3b6b6088bfff547ac1b>

Reply:
Thanks, those commits look great!